README.adoc

```diff
@@ -19,6 +19,21 @@
 
 = MicroProfile Health
 
+'''
+== How to contribute
+
+Do you want to contribute to this project? Here is what you can do:
+
+* Fork the repository, make changes, then do a pull request.
+* https://github.com/eclipse/microprofile-health/issues[Create or fix an issues].
```

Comment:
@phillip-kruger sorry, I missed this previously, noticed in different MP spec - "an issues" should be "an issue" or just "issues".

Reply:
Fixed